app/docker/views/containers/create/createContainerController.js

```diff
@@ -622,23 +698,13 @@ angular.module('portainer.docker').controller('CreateContainerController', [
           loadFromContainerImageConfig(d);
           loadFromContainerResources(d);
           loadFromContainerCapabilities(d);
+          loadConfigHealthcheck();
         })
         .catch(function error(err) {
           Notifications.error('Failure', err, 'Unable to retrieve container');
         });
     }
 
-    function loadFromContainerLogging(config) {
```

Comment:
Note this function is moved before loadFromContainerSpec() to avoid IDE warnings about the undefined function. Works out at runtime, but things should be declared before used in JS (and C...and life in general...).